app/types/nonCr3.ts

```diff
@@ -0,0 +1,50 @@
+import { z } from "zod";
+
+/**
+ * Validation schema for a non-CR3 record uploaded through the UI
+ */
+export const nonCr3UploadSchema = z.object({
```

Comment:
Zod is back! and it's making quick work of a [bunch of validation code](https://github.com/cityofaustin/vision-zero/blob/e00391d9040bf6a4154759535a7bef2487665465/editor/src/views/UploadNonCr3/UploadNonCR3.js#L297) we used in the old editor.